### PR TITLE
Blog post: secure remote access makes remote work a win-win

### PR DESCRIPTION
**Pull request by jefferenced**
Review & approve

### CODE REVIEW

**Review by jamilbk — APPROVED** [review]

**Comment by jamilbk** [comment]
@jefferenced Fixed the build so you should be able to preview on Vercel now and leave comments.

**Comment by jamilbk** [comment]
@AndrewDryga Looks like this blog post slowed down our throughput 😆. Any idea why the perf test is failing? Don't see anything obvious in the logs.

File: website/src/app/blog/secure-access/readme.mdx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+**Universal access:** With a secure remote access solution in place, authorized
+users can access a company network, resources, and shared services on any
+supported device. This means employees who are traveling, at home, or working
+from the coffee shop down the street always have access to the resources they
+need to do their job.
+
+**Device protection:** Employees at most organizations use lots of different
+devices — whether it’s email or meetings on a smartphone, writing code on a
+desktop, or updating reviewing PR’s on a laptop. An effective remote access
```
Comment (jefferenced) [inline website/src/app/blog/secure-access/readme.mdx]: Delete "updating"

File: website/src/app/blog/secure-access/readme.mdx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+Remote work (made possible by secure remote access, which helps employees get
+their work done safely) benefits everyone:
+
+**Save money:** Securing remote access benefits employees and employers alike,
+with cost savings across the board for supporting remote employees according to
+a Global Workplace Analytics (GWA)
+[report](https://globalworkplaceanalytics.com/resources/costs-benefits). This
+report indicates that compared to onsite employees, remote employees save
+thousands per year, and employees save around $10,000 for each remote employee.
```
Comment (jefferenced) [inline website/src/app/blog/secure-access/readme.mdx]: should be ", and employers"

**Review by jefferenced — COMMENTED** [review]
Small changes listed inline.

**Comment by AndrewDryga** [comment]
@jamilbk there are issues with buffers and Gabi js looking into them, so tests are not very reliable for now

File: website/src/app/blog/secure-access/readme.mdx (hunk 2 above)
Comment (jamilbk) [reply website/src/app/blog/secure-access/readme.mdx]: Fixed

**Pull request merged**